.env

```
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_API_HOST=
-NEXT_PUBLIC_APP_NAME=
+NEXT_PUBLIC_API_HOST=http://192.168.2.251:1337/api
+NEXT_PUBLIC_APP_NAME=quartz
```

remover variaveis ambientes

```suggestion
NEXT_PUBLIC_API_HOST=
NEXT_PUBLIC_APP_NAME=
```